src/assets/i18n/fr.json5

```diff
@@ -1614,55 +1597,43 @@
+  "browse.metadata.srsc.breadcrumbs": "Parcourir par catégorie",
 
   // "browse.metadata.srsc.tree.description": "Select a subject to add as search filter",
-  // TODO New key - Add a translation
-  "browse.metadata.srsc.tree.description": "Select a subject to add as search filter",
+  "browse.metadata.srsc.tree.description": "Sélectionnez un sujet à ajouter comme filtre de recherche",
 
   // "browse.metadata.nsi.breadcrumbs": "Browse by Norwegian Science Index",
-  // TODO New key - Add a translation
-  "browse.metadata.nsi.breadcrumbs": "Browse by Norwegian Science Index",
+  "browse.metadata.nsi.breadcrumbs": "Parcourir par Norwegian Science Index",
 
   // "browse.metadata.nsi.tree.description": "Select an index to add as search filter",
-  // TODO New key - Add a translation
-  "browse.metadata.nsi.tree.description": "Select an index to add as search filter",
+  "browse.metadata.nsi.tree.description": "Sélectionnez un index à ajouter comme filtre de recherche",
 
   // "browse.metadata.title.breadcrumbs": "Browse by Title",
   "browse.metadata.title.breadcrumbs": "Parcourir par Titre",
 
   // "browse.metadata.map": "Browse by Geolocation",
-  // TODO New key - Add a translation
-  "browse.metadata.map": "Browse by Geolocation",
+  "browse.metadata.map": "Parcourir par géolocalisation",
 
   // "browse.metadata.map.breadcrumbs": "Browse by Geolocation",
-  // TODO New key - Add a translation
-  "browse.metadata.map.breadcrumbs": "Browse by Geolocation",
+  "browse.metadata.map.breadcrumbs": "Parcourir par géolocalisation",
 
   // "browse.metadata.map.count.items": "items",
-  // TODO New key - Add a translation
   "browse.metadata.map.count.items": "items",
 
   // "pagination.next.button": "Next",
-  // TODO New key - Add a translation
-  "pagination.next.button": "Next",
+  "pagination.next.button": "Suivant",
 
   // "pagination.previous.button": "Previous",
-  // TODO New key - Add a translation
-  "pagination.previous.button": "Previous",
+  "pagination.previous.button": "Précédent",
 
   // "pagination.next.button.disabled.tooltip": "No more pages of results",
-  // TODO New key - Add a translation
-  "pagination.next.button.disabled.tooltip": "No more pages of results",
+  "pagination.next.button.disabled.tooltip": "Plus aucune page de résultats",
 
   // "pagination.page-number-bar": "Control bar for page navigation, relative to element with ID: ",
-  // TODO New key - Add a translation
-  "pagination.page-number-bar": "Control bar for page navigation, relative to element with ID: ",
+  "pagination.page-number-bar": "Barre de contrôle pour la page de navigation, relative à l'élément ayant l'identifiant : ",
 
-  // "browse.startsWith": ", starting with {{ startsWith }}",
-  // TODO New key - Add a translation
-  "browse.startsWith": ", starting with {{ startsWith }}",
+  // "browse.startsWith": ": "starting with {{ startsWith }}",
+  "browse.startsWith": ": "Commencant par {{ startsWith }}",
```

Comment:
This line is throwing the lint errors because it's invalid.  There's an extra quote (`"`) before "Commencant" that shouldn't be there. 

It also looks like you accidentally changed the example text as well in the line above this, as you've changed the comma into a colon.

The correct both issues, I think this should be:
```
// "browse.startsWith": ", starting with {{ startsWith }}",
"browse.startsWith": ", Commencant par {{ startsWith }}",
```